helpers/init_centos.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
```

Comment:
Is this script copied from the project factory? Do we need a copy here? If not, can we drop this file?

Reply:
Will be dropped in the next release